src/model/formats/format-utils.ts

```diff
@@ -73,7 +73,7 @@ export class FormatUtils {
       // Previous item was a title-like entry (detail or check item with note)
       (titleLikeItems.includes(lastItemEFIS.type) && lastItemEFIS.indent < itemEFIS.indent) ||
       // Previous item was an indented note-like entry (multiline note)
-      ([...FormatUtils.CHECKLIST_ITEM_PREFIXES.keys()].includes(lastItemEFIS.type) &&
+      ([...FormatUtils.CHECKLIST_ITEM_PREFIXES.keys(), ChecklistItem_Type.ITEM_SPACE].includes(lastItemEFIS.type) &&
```

Comment:
Consider just adding ITEM_SPACE to that map? That'd also make the logic on foreflight-writer.ts simpler.

Reply:
Removed, see last comment in #195.